pr_agent/git_providers/github_provider.py

```diff
@@ -75,9 +78,16 @@ def publish_inline_comment(self, body: str, relevant_file: str, relevant_line_in
         if position == -1:
             if settings.config.verbosity_level >= 2:
                 logging.info(f"Could not find position for {relevant_file} {relevant_line_in_file}")
+            subject_type = "FILE"
         else:
-            path = relevant_file.strip()
-            self.pr.create_review_comment(body=body, commit_id=self.last_commit_id, path=path, position=position)
+            subject_type = "LINE"
+        path = relevant_file.strip()
+        # placeholder for future API support (already supported in single inline comment)
```

Comment:
Consider handling the case where the 'subject_type' is 'FILE'. Currently, the code does not handle this case, which might cause issues when trying to publish the comment. [medium]